tensorflow/compiler/mlir/xla/transforms/lhlo_legalize_to_affine.cc

```diff
@@ -31,6 +31,61 @@ namespace mlir {
 namespace xla_lhlo {
 namespace {
 
+template<typename LhloOpTy>
+struct DotOpConverter : public OpRewritePattern<LhloOpTy>{
+  using OpRewritePattern<LhloOpTy>::OpRewritePattern;
+
+// Only Rank 2 tensors
```

Comment:
All comments are to be terminated with a period. Consider making it more descriptive.
```
// TODO: Supports only rank-2 tensors for now. 
``